File: samples/sample.java
public class Rectangle {
    private int width;
    private int height;

    public Rectangle(int width, int height) {
        this.width = width;
        this.height = height;
    }

    public int getArea() {
        return this.width * this.height;
    }

    public int getPerimeter() {
        return 2 * (this.width + this.height);
    }

    public static void main(String[] args) {
        Rectangle rect = new Rectangle(5, 10);
        System.out.println("Area: " + rect.getArea());
        System.out.println("Perimeter: " + rect.getPerimeter());
    }
}

File: samples/sample.c
#include <stdio.h>

typedef struct {
    int width;
    int height;
} Rectangle;

int getArea(Rectangle r) {
    return r.width * r.height;
}

int getPerimeter(Rectangle r) {
    return 2 * (r.width + r.height);
}

int main() {
    Rectangle rect = {5, 10};
    
    printf("Area: %d\n", getArea(rect));
    printf("Perimeter: %d\n", getPerimeter(rect));
    
    return 0;
}